python/fusion_engine_client/analysis/data_loader.py

```diff
@@ -282,7 +287,7 @@ def _read(self,
               return_numpy: bool = False, keep_messages: bool = False, remove_nan_times: bool = True,
               time_align: TimeAlignmentMode = TimeAlignmentMode.NONE,
               aligned_message_types: Union[list, tuple, set] = None,
-              quiet: bool = False) \
+              quiet: bool = False, source_ids: List[int] = [0]) \
```

Comment:
You need to document this argument, but you should move it up with `message_types` and `time_range` (and then add the documentation in the right place). The arguments here are organized into what they do: filter the data, control the returned values, etc.

Also, this should be `Optional` and should default to `None`.